### PR TITLE
Surya/core 9539/rm log filename

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

**Comment by maxtaco** [comment]
@heronhaye are you sure that systemd doesn't go through "auto-fork"?  In the old linux days, when you first ran the CLI, it spawned the daemon in the background if it didn't already exist. Then we passed through the logfile, I believe...

**Comment by heronhaye** [comment]
hey @taruti, we were trying to get rid of the `--log-file-name` in the systemd unit file and found that we needed to call `RotateLog` after setting the default logfile for logging to work. Does this look okay?

File: go/libkb/env.go [hunk 1]
```diff
@@ -1250,6 +1251,21 @@ func (e *Env) GetLogFile() string {
...
 }
 
+func (e *Env) GetUseDefaultLogFile() bool {
+	return e.GetBool(false,
+		e.cmd.GetUseDefaultLogFile,
+		func() (bool, bool) { return e.getEnvBool("KEYBASE_USE_DEFAULT_LOG_FILE") },
+	)
+}
+
+// func (e *Env) GetDisplayRawUntrustedOutput() bool {
```
Comment (maxtaco) [inline go/libkb/env.go]: what's this?

**Comment by eli-schwartz** [comment]
Thanks!

**Pull request merged**